src/api/app/views/webui2/webui/webui/_collapsible_text.html.haml

```diff
@@ -1,8 +1,8 @@
 - if text.present?
   .obs-collapsible-textbox
     - if text.length > threshold[:text_length] || text.lines.count > threshold[:line_count]
-      .obs-collapsible-text.obs-collapsed
-        = simple_format(text, class: 'm-0 p-0')
-      %a.obs-collapsible-link.obs-collapsed.text-center.d-block{ title: 'Show more' }
+      .obs-collapsible-text.obs-collapsed{ class: "line_count_#{threshold[:line_count]}" }
```

Comment:
My concern with this is that anyone who is going use this partial would have to know about how this works.
Two possible solutions that would come to my mind would be:
* Document this in our pattern library
* Make this a "switch", like small collapsible, large collapsible

Reply:
What probably also would work is to write a [custom sass function](https://vanseodesign.com/css/custom-sass-functions/) for this. I never did this, so I can't tell how much effort this would and if that is a sound solution.

Reply:
@bgeuken, I will have a look into custom sass function